lib/data_update_scripts/20200904132553_remove_draft_articles_with_duplicate_feed_source_url.rb

```diff
@@ -6,7 +6,7 @@ def run
       # This statement deletes all draft articles in excess found to be duplicate over feed_source_url,
       # excluding those whose body_markdown is different from the other duplicate occurrences
       result_same_body = ActiveRecord::Base.connection.execute(
-        <<~SQL,
+        <<~SQL.squish,
```

Comment:
Rubocop fix you can learn more about it in this PR: https://github.com/forem/forem/pull/10237

Reply:
TIL `.squish` -- love it!